components/TextForageView.tsx

```diff
@@ -631,26 +733,7 @@ export function TextForageView({ collectionId }: { collectionId?: string }) {
             {/* </YStack> */}
             <YStack position="absolute" zIndex={1} right="$1.5" bottom="$2">
               <StyledButton
-                onPress={async () => {
-                  if (existingMedias.size > 0) {
-                    Alert.alert(
-                      "Duplicate Media",
-                      "You've already added some of this media. Are you sure you want to add it again?",
-                      [
-                        {
-                          text: "Cancel",
-                          style: "cancel",
-                        },
-                        {
-                          text: "Add Anyway",
-                          onPress: () => onSaveResult(),
-                        },
-                      ]
-                    );
-                  } else {
-                    onSaveResult();
-                  }
-                }}
+                onPress={onSend}
```

Comment:
hmm way too big of a difference. switch back to using the onSaveResult and maybe instead pass on the collectionIds as an argument? it's only used in this one context so changing it should be fine.